packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -94,22 +94,23 @@ export async function produceBlockBody<T extends BlockType>(
   this: BeaconChain,
   blockType: T,
   currentState: CachedBeaconStateAllForks,
-  {
-    randaoReveal,
-    graffiti,
-    slot: blockSlot,
-    feeRecipient: requestedFeeRecipient,
-    parentSlot,
-    parentBlockRoot,
-    proposerIndex,
-    proposerPubKey,
-  }: BlockAttributes & {
+  blockAttr: BlockAttributes & {
     parentSlot: Slot;
     parentBlockRoot: Root;
     proposerIndex: ValidatorIndex;
     proposerPubKey: BLSPubkey;
+    phase0BlockBody?: phase0.BeaconBlockBody;
   }
 ): Promise<{body: AssembledBodyType<T>; blobs: BlobsResult; executionPayloadValue: Wei}> {
+  const {
+    slot: blockSlot,
+    feeRecipient: requestedFeeRecipient,
+    parentSlot,
+    parentBlockRoot,
+    proposerIndex,
+    proposerPubKey,
+    phase0BlockBody,
```

Comment:
```suggestion
    commonBlockBody,
```
lets not call it phase0 block body since there can be common fields like (blsToExecution) and its type can be Omit<allForks.BeaconBlockBody,"executionPayload">